client/app/telecom/sms/sms/templates/telecom-sms-sms-templates.html

```diff
@@ -67,23 +68,23 @@ <h1 data-translate="sms_sms_templates_title"></h1>
                     aria-labelledby="btn-sms-templates-actions"
                     data-uib-dropdown-menu>
                     <li role="menuitem">
-                        <i class="ovh-font ovh-font-add-circle" aria-hidden="true"></i>
+                        <span class="ovh-font ovh-font-add-circle" aria-hidden="true"></span>
```

Comment:
This dropdown could be a `<extra-top>` (ref: http://master.ui-kit.ovh/#!/oui-angular/datagrid)?

Actually all `<span class="ovh-font ovh-font-*">…` aren't displayed.